### DISCUSSION TITLE
High Mysql Usage

### DISCUSSION THREAD

**Discussion by Undrego** [jchristn/Less3#6] [Q&A]
Good Day. Firstly thank you for a great s3 server

I have one small issue i cant seem to get past.

I setup the less3 server on a ubuntu box with mysql as db. this works great but when traffic picks up then mysql load skyrockets to 100% and when viewing mysql processes this is displayed.

![image](https://github.com/jchristn/Less3/assets/82903754/e977ce98-993e-4693-90f0-5b19808647a0)

Can you give me any advise on how to resolve this. Apologies for the bother and thank you once again

**Comment by jchristn** [comment]
Hi, sorry I'm just now seeing this.  I'd recommend adding indices to each field in each table that are often used in queries, essentially anything with a GUID and certain other keys.
